### PR TITLE
Updated Supported Versions tables for Django 4.1.

### PR DESCRIPTION
**Pull request by carltongibson**
To be merged on Aug 3rd. 

@felixxm I added a second commit in 3b26a1dff0bfec06305e207f9db82c281e890fe0 — Let me know your 👍/ 👎 😉

### CODE REVIEW

File: djangoproject/templates/releases/download.html [hunk 1]
```diff
@@ -75,6 +75,11 @@ <h2 id="supported-versions">Supported Versions</h2>
         <th>End of mainstream support<sup><a href="#ft1">1</a></sup></th>
         <th>End of extended support<sup><a href="#ft2">2</a></sup></th>
       </tr>
+      <tr>
+        <td>4.1</td>
+        <td>April 2023</td>
+        <td>December 2023</td>
```
Comment (felixxm) [inline djangoproject/templates/releases/download.html]: This should have 4 columns, `<td>{% get_latest_micro_release '4.1' %}</td>` is missing.

Comment (carltongibson) [reply djangoproject/templates/releases/download.html]: Good spot. Thanks. 

I got distracted by the indentation 🤦‍♀️

**Comment by carltongibson** [comment]
Hang on... will rebase.

File: djangoproject/templates/releases/download.html [hunk 2]
```diff
@@ -69,24 +69,30 @@ <h2 id="supported-versions">Supported Versions</h2>
...
-      </tr>
+    <tr>
+      <th>Release Series</th>
+      <th>Latest Release</th>
+      <th>End of mainstream support<sup><a href="#ft1">1</a></sup></th>
+      <th>End of extended support<sup><a href="#ft2">2</a></sup></th>
+    </tr>
+    <tr>
+      <td>4.1</td>
+      <td>{% get_latest_micro_release '4.1' %}</td>
```
Comment (felixxm) [inline djangoproject/templates/releases/download.html:80]:
```suggestion
      <td>{% get_latest_micro_release '4.1' %}</td>
```

**Review by felixxm — APPROVED** [review]
@carltongibson Thanks :+1:

**Pull request merged**